x-pack/plugin/src/test/resources/rest-api-spec/api/ml.get_data_frame_analytics.json

```diff
@@ -14,6 +14,11 @@
         }
       },
       "params": {
+        "allow_no_match": {
+          "type": "boolean",
+          "required": false,
+          "description": "Whether to ignore if a wildcard expression matches no data frame analytics. (This includes `_all` string or when no data frame analytics have been specified)"
```

Comment:
I think it would be useful to add `"default": true` here.  Same in `ml.get_data_frame_analytics_stats.json`.